openedx/features/learner_profile/static/learner_profile/templates/section_two.underscore

```diff
@@ -2,9 +2,9 @@
     <div class="field-container"></div>
     <% if (!showFullProfile) { %>
         <% if(ownProfile) { %>
-            <span class="profile-private--message" tabindex="0"><%- gettext("You are currently sharing a limited profile.") %></span>
```

Comment:
"A value of 0 indicates that the element should be placed in the default navigation order. This allows elements that are not natively focusable (such as <div>, <span>, <p>, and <a> with no href) to receive keyboard focus. Links and form controls are best for almost all interactive elements, but tabindex="0" allows other elements to be focusable and able to trigger interaction." 

There doesn't seem to be a reason to allow tabbing into this span, although @andy-armstrong, do you have any thoughts as to an accessibility reason for this? Screen readers can still access the div, and there is no enter/spacebar behavior, this seems quirky. 

See the image below. That is with the tab-index.

<img width="870" alt="screen shot 2017-08-07 at 4 22 39 pm" src="https://user-images.githubusercontent.com/3861052/29044285-b40f82f0-7b8c-11e7-94e5-1f289fb9b25a.png">